Gemfile.lock

```diff
@@ -13,7 +13,7 @@ GIT
 PATH
   remote: .
   specs:
-    shiftcommerce-rails (0.4.6)
```

Comment:
It was, yes – forgot to `bundle` after bumping version before. I'll remove the current `v0.5.1` and replace it with this once this PR is merged :)